docs/installation/server-config/security.rst

```diff
@@ -155,30 +160,34 @@ that includes the authenticated KSQL user.
 .. tip:: For more information about ACLs see :ref:`kafka_authorization` and for more information about interactive and
 non-interactive queries, see :ref:`restrict-ksql-interactive`.
 
+.. _config-security-ksql-acl-interactive:
+
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 Interactive KSQL clusters
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Kafka clusters that use ACLs to control access to resources require clients to authenticate as a particular user. Interactive
-KSQL clusters require that this authenticated KSQL user has open access to create, read, write, delete topics, and use any
-consumer group.
+:ref:`Interactive KSQL clusters <restrict-ksql-interactive>`, (which is the default configuration),
+require that the authenticated KSQL user has open access to create, read, write, delete topics, and use any consumer group:
 
-:ref:`Interactive KSQL clusters <restrict-ksql-interactive>` require these ACLs:
+Interactive KSQL clusters require these ACLs:
 
 - Permission for the ``DESCRIBE_CONFIGS`` operation on the ``CLUSTER`` resource type.
 - Permission for the ``CREATE`` operation on the ``CLUSTER`` resource type.
 - Permissions for ``DESCRIBE``, ``READ``, ``WRITE`` and ``DELETE`` operations on all ``TOPIC`` resource types.
-- Permissions for ``DESCRIBE`` and ``READ`` operations  on all ``GROUP`` resource types.
+- Permissions for ``DESCRIBE`` and ``READ`` operations on all ``GROUP`` resource types.
 
 It is still possible to restrict the authenticated KSQL user from accessing specific resources using ``DENY`` ACLs. For
 example, you can add a ``DENY`` ACL to stop KSQL queries from accessing a topic that contains sensitive data.
 
+.. _config-security-ksql-acl-headless:
+
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Non-Interactive (headless) KSQL clusters
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-:ref:`Non-interactive KSQL clusters <restrict-ksql-interactive>` can be run with much more restrictive ACLs, though it
-currently requires a bit of effort to determine what ACLs are required. This will be improved in future KSQL releases.
+:ref:`Non-interactive KSQL clusters <restrict-ksql-interactive>` can be run with much more restrictive ACLs
```

Comment:
suggest: `Because the list of queries are known ahead of time, you can run non-interactive KSQL clusters with more restrictive ACLs.`

Reply:
Done